extensions/amp-sidebar/0.1/storybook/Basic.amp.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// To do: how to add CSS
+//import '!style-loader!css-loader!./Basic-styles.css';
```

Comment:
See https://github.com/ampproject/amphtml/pull/27800

Reply:
Thanks, I'll add a separate PR to include more use cases with CSS

Reply:
The CSS pull request is approved.